### ISSUE TITLE
html unparse produces wrong results if two entities are close

### ISSUE THREAD

**Issue by Princic-1837592** [pyrogram/pyrogram#870]
## Checklist
- [x] I am sure the error is coming from Pyrogram's code and not elsewhere.
- [x] I have searched in the issue tracker for similar bug reports, including closed ones.
- [x] I ran `pip3 install -U https://github.com/pyrogram/pyrogram/archive/master.zip` and reproduced the issue using the latest development version.

## Description
When calling ```message.text.html``` I get a wrong result in case two entities are consecutive

``` python
msg = await client.send_message(chat_id, '<code>entity 1</code><code>entity 2</code>')
print(msg.text.html)
# expected: <code>entity 1</code><code>entity 2</code>
# actual result: <code>entity 1<code></code>entity 2</code>
```
This happens because at line 179 of ```html.py``` the list of entities is sorted by offset, but two consecutive entities have same offset: one ends and the other starts at the same point in the string so they are sorted as equal offset value, but have different tag (one closing and one opening)

## Traceback
No traceback raised, just wrong html format

**Comment by Princic-1837592** [comment]
Fixed in this pull request https://github.com/pyrogram/pyrogram/pull/872

**Comment by delivrance** [comment]
Fixed in #885. Discussion at #872.
